apollo-router/src/notification.rs

```diff
@@ -510,7 +510,11 @@ where
 
         match Pin::new(&mut this.msg_receiver).poll_next(cx) {
             Poll::Ready(Some(Err(BroadcastStreamRecvError::Lagged(_)))) => {
-                tracing::info!(monotonic_counter.apollo_router_skipped_event_count = 1u64,);
+                u64_counter!(
+                    "apollo_router_skipped_event_count",
+                    "Amount of times the router missed subscription events",
```

Comment:
I'm quite unsure about this description

Reply:
Right. I was trying to make it meaningful to users. I could only find this being used for subscriptions, but the way I wrote it is probably not what actually happens. How would you explain it to someone who is running the router and doesn't know much about the implementation details?

Reply:
```suggestion
                    "Amount of events dropped from the internal message queue",
```
We could just say it's "internal" if it's not something easily explained to a user

Reply:
Your last description LGTM